### PR TITLE
Refine EA error messages

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/73866

This PR updates EA error messages as per doc: https://docs.google.com/document/d/1ultz-_NklnLPC_7yLg_WAKaW9shBITB-44u4vkonyfU/edit

Errors that might be due to a client timeout will also get a link added to refine the import range.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task &quot;@luca add the Knowledgebase link here!&quot; | [EA_Messages.php:36](https://github.com/moderntribe/the-events-calendar/blob/6679a2aa4f1d16c56c6128d2878323d4b65d31c8/src/Tribe/REST/V1/EA_Messages.php#L36)

> via jenkins/codesniffer-phpcs

File: src/Tribe/REST/V1/EA_Messages.php [hunk 1]
```diff
@@ -4,21 +4,44 @@ class Tribe__Events__REST__V1__EA_Messages extends Tribe__Events__REST__V1__Mess
...
+			'tec-rest-api-archive-header-error',
+			'tec-rest-api-archive-missing-total',
+			'tec-rest-api-archive-bad-total',
+			'tec-rest-api-archive-page-error',
+			'tec-rest-api-archive-page-empty',
+			'tec-rest-api-single-event-error',
+			'tec-rest-api-single-event-empty',
+		);
+
+		// @todo @luca add the Knowledgebase link here!
```
Comment (barryhughes) [inline src/Tribe/REST/V1/EA_Messages.php]: Is the plan to merge and update later (I feel like you mentioned this on the weekly review meeting, but I can't remember what you said). Also, I wonder if we could create and use an `m.tri.be` link and simply update the destination URL later, once the KB post exists

Comment (lucatume) [inline src/Tribe/REST/V1/EA_Messages.php]: No, I'd like this to go in with the right link; I will wait to have that final link before merging this.

**Comment by lucatume** [comment]
Putting this on hold to wait for the knowledge base link.

**Pull request merged**